server/tests/testarea.py

```diff
@@ -0,0 +1,39 @@
+from gqlComms import listHistograms, createHistogram, deleteHistogram,getHistogram,getHistograms
+import numpy as np
+import datetime
+import argparse
+
+def test_createHist():
+    NUM = 100
+    LENGTH = 50
+    LOW = 0
+    HIGH =1000
+    rng = np.random.default_rng()
+    now = datetime.datetime.now()
+    runHeader = now.strftime("%Y%m%d_run")
+
+    params ={
+            'id': 100,
```

Comment:
As discussed, you should check the database to make sure you are not overriding an existing histogram, and if you are, then you should pick a new ID